qiskit_nature/results/protein_folding_tools/protein_xyz.py

```diff
@@ -0,0 +1,135 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""An auxiliary class that gets the coordinates of the aminoacids of a molecule
+ in ProteinFoldingResult """
+from typing import Union, List, Optional
+import numpy as np
+
+from qiskit_nature.problems.sampling.protein_folding.peptide.peptide import Peptide
+
+
+class ProteinXYZ:
```

Comment:
```suggestion
class ProteinShapeFileGen
```
Do we have better ideas?